### ISSUE TITLE
Twitter: Annouce Wildfly 23 Final - MicroProfile 4.0 compatible

### ISSUE THREAD

**Issue by radcortez** [microprofile/microprofile-marketing#451]
Can we please use the MP Twitter handle to announce that Wildfly 23 Final is now compatible with MicroProfile 4.0? More information here: https://www.wildfly.org/news/2021/03/11/WildFly23-Final-Released/.

Thanks!

**Comment by aeiras** [comment]
Of course, we ought to submit tickets for all the compatible news Roberto. 
 Write the tweet to enable that push. 

While at it, please submit a new ticket requesting that a design card template is created to announce this and future compatible implementations. Gracias RC!

**Comment by tomjenkinson** [comment]
When it is announced please can you use the casing "WildFly" for the word?

**Comment by aeiras** [comment]
Hola Tom, 

Please submit the Twitter description. Not submitting it today  will delay the media announcement to next week. 

Keep in mind that tweets allow for final review before publication yet implementation teams need to work & help improve the marketing process of pushing media content. This action is similar to speaker engagement call to action cards & push that is in place & successful for years 💛

**Comment by tomjenkinson** [comment]
Thanks @aeiras - I invite @bstansberry (WildFly project lead) to provide the twitter description

**Comment by bstansberry** [comment]
Hi @aeiras -- thanks for the info! I'll post something here later this morning; in meetings for a while longer.

**Comment by bstansberry** [comment]
@aeiras How's this?

"@WildFlyAS 23 includes support for @MicroProfileIO 4.0! Learn more about MicroProfile in WildFly at https://t.co/19UISbzopU?amp=1"

**Comment by aeiras** [comment]
Hey Brian, 

Published tweet [here](https://twitter.com/MicroProfileIO/status/1370432229051047938?s=20)

Can you please drop the Eclipse from the MicroProfile name in the blog and help also adjust the documentation where the Eclipse is inserted into the MicroProfile trademark. 
Under the Working Group creation, we were able to revert to the project original and desired name, just MicroProfile. 

Thank you for that blog update. I understand that documentation might take sometime to fix. If help is needed, reach out the MP forum for volunteers to help with that. :)

**Comment by aeiras** [comment]
Oh and Brian, I highly recommend you and the WildFly team crosspost current and future technical MP content to the MicroProfile blog.  Check out the how directly in the call to action, Become an Author: https://microprofile.io/blog/

**Comment by bstansberry** [comment]
Thanks, Amelia! I've updated the blog. The docs change, at least in the main WildFly docs, doesn't look too hard;  there aren't too many uses.

**Comment by aeiras** [comment]
I don't see the changes yet, with much focus on write up below "new features" section. 

Very happy to hear that the update across the WildFly ecosystem to match MP Trademark won't be a night. :) 

Thank you to the 3 of you for helping launch the MP Compatible Implementation media push. You 3 are wonderful! A project will be created within this space to enable others to own it. You have helped create **project 7** in a bit. The design is also necessary, a nice feature to have for later.

**Comment by bstansberry** [comment]
I updated the docs under https://docs.wildfly.org/23/ to change 'Eclipse MicroProfile' to 'MicroProfile'. The source that will generate those for later releases will be updated too once my PR is merged.

**Comment by aeiras** [comment]
Your actions amazing and you truly are a flash, Brian! 

Pull me and anyone on the branding/business side via a git ticket anytime. It is your job to push via their MP media channels the awesomeness accomplished by your team. I believe that feeding "rock moved", this is how we did it sharing works and inspire us to join the fun. 

Abrazos,

**Comment by aeiras** [comment]
Tom and others, 

You have enabled the creation of project 7 about MP Implementation media Announcement.  You are most welcomed to improve the requirements of the call to action if you wish to do so. Stay safe and awesome! 

https://github.com/eclipse/microprofile-marketing/projects

**Comment by tomjenkinson** [comment]
Thank you @aeiras!
